core/src/avm2/globals/flash/sensors/Accelerometer.as

```diff
@@ -0,0 +1,7 @@
+package flash.sensors {
+    public class Accelerometer {
```

Comment:
This appears to be a a version-gated class- could you insert the line `[API("667")]` before `public class Accelerometer`?